infra/scripts/language/agents/triage_agent.py

```diff
@@ -0,0 +1,38 @@
+import json
+from azure.ai.agents.models import OpenApiTool
+from utils import bind_parameters
+
+def get_openapi_tool(filename, name, description, config, auth):
+    with open(filename, "r") as f:
+        spec = json.loads(bind_parameters(f.read(), config))
+    return OpenApiTool(
```

Comment:
nit - you should put the return in the `open` context, and raise an error if you don't return there

e.g.

```
with open(...):
    spec = ...
    return ...

raise ValueException("Unable to return ...")
```